infrastructure/terraform/modules/typescript-build-zip/null_resource_typescript_build.tf

```diff
@@ -5,6 +5,6 @@ resource "null_resource" "typescript_build" {
 
   provisioner "local-exec" {
     working_dir = var.source_code_dir
-    command     = "npm ci && npm run build"
+    command     = "rm -rf ${var.output_dir} && npm ci && npx esbuild --bundle --minify ${var.include_sourcemaps ? "--sourcemap=inline" : ""} --target=es2020 --platform=node --outdir=${var.output_dir} ${join(" ", var.entrypoints)}"
```

Comment:
Would it be nicer for the TF to call a bash script to allow this to be formatted nicely and to make it easier to extend in the future?

Reply:
I think that would be more complicated than it's worth now - could do it in the future if this command did get more complicated. But I can't really see it needing to get more complex than this. And putting it in a bash script adds complexity about which directory it's running from, parsing arguments etc. Whereas it's working fine like this. 😅